src/modules/home/components/reservation-card.tsx

```diff
@@ -0,0 +1,198 @@
+import React, { useState, useEffect } from 'react';
+import Box from '@mui/material/Box';
+import Typography from '@mui/material/Typography';
+import TextField from '@mui/material/TextField';
+import Button from '@mui/material/Button';
+import MenuItem from '@mui/material/MenuItem';
+import Divider from '@mui/material/Divider';
+import { useTheme } from '@mui/material/styles';
+
+interface ReservationCardProps {
+  pricePerNight: number;
+  cleaningFee: number;
+  serviceFee: number;
+  maxGuests: number;
+  availableFrom: string;
+  availableTo: string;
+}
+
+const styles = {
+  cardContainer: (theme: any) => ({
+    border: '1px solid',
+    borderColor: theme.palette.primary.main,
+    borderRadius: '8px',
+    padding: '16px',
+    maxWidth: '400px',
+    boxShadow: '0px 2px 4px rgba(0, 0, 0, 0.1)',
+  }),
+  button: (theme: any, isEnabled: boolean) => ({
+    backgroundColor: isEnabled ? theme.palette.secondary.main : theme.palette.action.disabled,
+    color: 'white',
+    fontWeight: 'bold',
+    marginBottom: '16px',
+    height: '56px',
+    '&:hover': {
+      backgroundColor: isEnabled ? theme.palette.secondary.dark : theme.palette.action.disabled,
+    },
+  }),
+  textField: {
+    flex: 1,
+  },
+  section: {
+    display: 'flex',
+    justifyContent: 'space-between',
+    marginBottom: '8px',
+  },
+};
+
+export const ReservationCard: React.FC<ReservationCardProps> = ({
+  pricePerNight,
+  cleaningFee,
+  serviceFee,
+  maxGuests,
+  availableFrom,
+  availableTo,
+}) => {
+  const theme = useTheme();
+
+  const [checkIn, setCheckIn] = useState<string>('');
+  const [checkOut, setCheckOut] = useState<string>('');
+  const [guests, setGuests] = useState<number>(1);
+  const [isButtonEnabled, setIsButtonEnabled] = useState<boolean>(false);
+
+  const getTodayDate = (): string => {
```

Comment:
Please move this function to shared utils. It may be used by other components as well